app/jobs/parse_csv_job.rb

```diff
@@ -86,7 +86,7 @@ def create_csvs()
         csv << hash.values
       end
     end
-    CSV.open("/Users/katepiette/ohana-api/data/services.csv", "wb" , options = { :quote_char=>" " }) do |csv|
+    CSV.open("/Users/katepiette/ohana-api/data/services.csv", "wb") do |csv|
```

Comment:
@superkat64 Change the path to /tmp/data/services.csv, remember that you need deploy the code and run the parser in staging/production in order to load the data provided by Joel

Reply:
Also, I can suggested that we need include the data parsed and not the function that you have created for parse that data, I mean, you have a Job for read a csv file in your computer and that function is used for created a lot of csv file used for Ohana, in my opinion, you only should uploaded the csv files that you have obtained in the result of parse the data, that is because the description of the ticket is: Parse Client data for API

Reply:
Okay, that sounds good to me.